management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -180,10 +188,21 @@ and the last section shows four individual metrics: Configurations without a Des
 {: .image-popup}
 ![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-overview.png)
 
+*Overview* shows key metrics regarding the health of jobs and flows. *Jobs Error Ratio/Flow Error Ratio* indicates the percentage of jobs that ended in error relative to successful ones, while and *Error 
+Jobs/Error Flows* presents the absolute number of jobs that concluded with errors. Additionally, since flows can also end with warnings, two metrics display the percentage and absolute number of flows with 
+warning. 
+
+Below, two graphs illustrate the health of flows and jobs over time. By hovering over a particular day on the graph, you can view more detailed data, including the number of flows or jobs that ended with 
+success, error, or warning on that day.
+
 #### Components
 {: .image-popup}
 ![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-components.png)
 
+This section provides a more detailed view of the errors. *Error Jobs PPU* displays the number of credits consumed by jobs that ended in error; you can drill down to see further details. *PPU Consumed by Error 
+Jobs* shows the trend of credit consumption over time. *Components Errors Trend* examines the errors, split by component type, over time. The last two graphs, *Components Errors* and *Components Error Ratio*, 
+present similar information on component errors—the first in absolute values and the second as a percentage.
+
 #### Configurations, flow notifications, and tables
 {: .image-popup}
 ![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-config.png)
```

Comment:
@hhanova - I am almost 100% sure I wrote this yesterday... but obviously not... sorry, could you please add this below to this part? 
This section provides information on health from the point of housecleaning. *Configurations without Description* provide the number and the list of configuration where no information on how these works are provided. *Flows without Error Notifications* shows how many flows ends up in the error and nobody finds out as there is no notification set up. *Unused Tables* and *Unused Configurations* show how many tables haven't had any import or export in the last 30 days, similarly for configurations - how many of them didn't have any job started in the last 30 days.